docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -184,7 +184,20 @@ Step 1: Download and Install the OpenVINO Core Components
       cd /opt/intel/openvino_2023.0.1
       sudo -E ./install_dependencies/install_openvino_dependencies.sh
 
-6. For simplicity, it is useful to create a symbolic link as below:
+6. (Optional) Install *numpy* Python Library:
+
+   .. note::
+
+      This step is required only when you decide to use Python API.
+
+   You can use the ``requirements.txt`` file from the ``opt/intel/openvino_2023.0.1/python/python.<x>`` folder:
```

Comment:
```suggestion
   You can use the ``requirements.txt`` file from the ``/opt/intel/openvino_2023.0.1/python`` folder:
```
in master we installing requirements.txt files into `opt/intel/openvino_2023.0.1/python` folder (no per-python version subfolders).